### ISSUE TITLE
Update the  java image used in the Docker stack

### ISSUE THREAD

**Issue by cescoffier** [vert-x3/vertx-stack#50]
Docker has decided to mark the _java:*_ images as deprecated (see https://hub.docker.com/_/java/). 

```
This image is officially deprecated in favor of the openjdk image, and will receive no further updates after 2016-12-31 (Dec 31, 2016). Please adjust your usage accordingly.

The image has been OpenJDK-specific since it was first introduced, and as of 2016-08-10 we also have an ibmjava image, which made it even more clear that each repository should represent one upstream instead of one language stack or community, so this rename reflects that clarity appropriately.
```

Our images are depending on these deprecated images. But we still have some time. As there are ongoing changes in the Docker containers planned for 3.4.0, I'm going to update the parent image in this version. The planned change are breaking changes anyway (see https://github.com/vert-x3/vertx-stack/pull/48).

The parent images will be:
- openjdk:8u102-jre (+)
- openjdk:8u92-jre-alpine (+)

(versions may be updated)

**Comment by cescoffier** [comment]
The fix has been applied in https://github.com/vert-x3/vertx-stack/pull/48 that is going to be merged once we start working on 3.4.0

**Comment by cescoffier** [comment]
Fixed in #48
